src/components/network/network-table.js

```diff
@@ -50,10 +71,136 @@ genotet.NetworkTable.prototype.create = function(table, edges) {
       {title: 'Weight', data: 'weight'},
       {title: '', data: 'added'}
     ],
+    select: {
+      style: 'os',
+      info: false
+    },
+    dom: '<"row"<"#table-btn.col-sm-12"B>>' +
+      '<"row">' +
+      '<"row"<"col-sm-5"l><"col-sm-7"f>>' +
+      '<"row"<"col-sm-12"tr>>' +
+      '<"row"<"col-sm-5"i><"col-sm-7"p>>',
+    buttons: [
+      {
+        text: 'Add',
+        className: 'btn btn-default',
+        action: function(e, dt, node, config) {
+          var selectedEdges = dt.rows({selected: true}).data();
+          var additionEdges = [];
```

Comment:
Umm. Probably because data() returns sth that is not a pure array. I think it would be necessary to annotate what data() returns. Otherwise it's hard to trace what the code is accessing from the table data().